nl.hjdskes.gcolor3.json

```diff
@@ -0,0 +1,45 @@
+{
+    "app-id": "nl.hjdskes.gcolor3",
+
+    "runtime": "org.gnome.Platform",
+    "runtime-version": "3.28",
+    "sdk": "org.gnome.Sdk",
+
+    "command": "gcolor3",
+
+    "finish-args": [
+        "--share=ipc",
+        "--socket=x11",
+        "--socket=wayland",
```

Comment:
Since this doesn't actually work on Wayland perhaps just remove this?

Reply:
What do I need that for? It's working fine for me in Wayland at the moment without dconf access.

Reply:
GTK on Wayland uses DConf.